lib/galaxy/config.py

```diff
@@ -697,7 +697,9 @@ def configure_logging( config ):
     # PasteScript will have already configured the logger if the
     # 'loggers' section was found in the config file, otherwise we do
     # some simple setup using the 'log_*' values from the config.
-    if not config.global_conf_parser.has_section( "loggers" ):
+    paste_configures_logging = config.global_conf_parser.has_section( "loggers" )
+    auto_configure_logging = string_as_bool( config.get( "auto_configure_logging", not paste_configures_logging ) )
```

Comment:
Because it is a better behavior or because that is the what the documentation in ``galaxy.ini.sample``. I think it is actually more useful the way it is, but it is not quite how it is describe in the sample.